presets/nest+next/config.yml

```diff
@@ -0,0 +1,33 @@
+# Which tags are related to this preset; used for branching the choices on preset wizard
+tags: [ 'Typescript' ]
+
+# Create defines the workflow for creating a new Project
+# where this preset can then be installed
+create:
+  - name: Creating new NestJS Application
+    actions:
+      - scripts:
+          - docker pull -q kooldev/node:16
+          - mkdir $CREATE_DIRECTORY
+          - kool docker kooldev/node:16 npx -y @nestjs/cli new -l Typescript -p npm $CREATE_DIRECTORY/backend
+          - kool docker kooldev/node:16 npx -y create-next-app@latest --ts --use-npm $CREATE_DIRECTORY/frontend
+
+preset:
+  - name: 'Copy basic config files'
+    actions:
+      - copy: docker-compose.yml
+      - copy: kool.yml
+      - merge: nest+next-docker-compose.yml
+        dst: docker-compose.yml
+      - recipe: pick-db
+      - recipe: pick-cache
+      - copy: env.dist
+        dst: .env.dist
+      - copy: env.dist
+        dst: .env
```

Comment:
The `copy` and `dst` for `env.dist` are duplicated.

Reply:
So this one is actually correct Steve - I am copying twice, once the actual .env.dist and the second time we actually create `.env`.

The project starts then with both `.env.dist` and `.env` in place already.